### DISCUSSION TITLE
Why does version 1.1.6 limit the total number of CPU threads?

### DISCUSSION THREAD

**Discussion by vanthoi** [Chia-Network/chia-blockchain#6016] [Ideas]
I have 16 cores and 32 threads. I use 2 threads for 8 parallel plots. After all this plot ends phase 1. I want to utilize the excess thread count to create 8 more plots. But I can only create 5 more halves. The other two threads are for the system.
While 8 plots have passed phase 1 we can take advantage of 4 unused threads. When creating a plot past phase 1, only one thread is used. Why not allow to take advantage of the remaining processing thread to create plots?

**Comment by WMark77** [comment]
Same issue.
